docs/platforms/apple/common/install/cocoapods.mdx

```diff
@@ -15,3 +15,5 @@ end
 ```
 
 Afterwards run `pod install`.
+
+In case you are using Xcode 14 or later, you might have to disable the "Enable User Script Sandbox" option in the target settings by setting it to `NO`. See [Troubleshooting](/platforms/apple/common/troubleshooting) for more information.
```

Comment:
`h`: You need to use `<PlatformLink to= ...` Just search for it in the codebase and you will see how to use it.